template/.github/workflows/ci.yml

```diff
@@ -170,3 +170,49 @@ jobs:
 
     - name: Run the functional test suite
       run: poetry run invoke test --suite=functional
+
+  build-docker-image:
+    name: Build Docker Image
+    needs: [static-analysis-checks, type-checking, run-system-tests, run-functional-tests]
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:14
+        env:
+          POSTGRES_USER: dev
+          POSTGRES_PASSWORD: dev_password
+          POSTGRES_DB: dev
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+            # Allows us to use the same env config used locally by mapping to port to
+            # the test runner container.
+            - 5432:5432
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+
+    - name: Install dependencies
+      run: pip install invoke
+
+    - name: Setup environment variables
+      run: |
+        cp example.env .env
+        echo 'DATABASE_URL=postgres://dev:dev_password@postgres:5432/dev'  >> .env
+
+    - name: Build service Docker Image
+      run: invoke build-image
+
+    - name: Run service Docker image
+      run: |
+        invoke run-image --options '--network ${{ job.container.network }} --env-file .env' --command check
```

Comment:
With passing `job.container.network` from [github action context](https://docs.github.com/en/actions/learn-github-actions/contexts#job-context) we can link the docker container to the postgres service.

The problem here is that when starting a new project, djnago takes `${{ job.container.network }}` a variable and converts it to an empty string. I tried `${{{{ job.container.network }}}}` to scape the conversion but it raises:
```
raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '{{ job.container.network' from '{{ job.container.network'
```
Do you have any idea to fix this issue?

Reply:
It resolved with usign `{% verbatim %}` 💚